zerver/openapi/zulip.yaml

```diff
@@ -6200,7 +6200,11 @@ paths:
             subscribed to appropriate channels or use a shared history
             search narrow with this endpoint.
 
-            **Changes**: In Zulip 9.0 (feature level 250), narrows gained support
+            **Changes**: In Zulip 9.0 (feature level 262), narrows gained support for a new
+            filter, `with`, which returns messages in the same channel and topic as that
+            of the operand of the filter, with the first unread message as anchor.
```

Comment:
You should redo this with some of the more precise language from the API design conversation.

Reply:
done!